examples/pie_and_polar_charts/pie_features.py

```diff
@@ -1,41 +1,103 @@
 """
-===============
-Basic pie chart
-===============
+==========
+Pie charts
+==========
 
-Demo of a basic pie chart plus a few additional features.
-
-In addition to the basic pie chart, this demo shows a few optional features:
-
-* slice labels
-* auto-labeling the percentage
-* offsetting a slice with "explode"
-* drop-shadow
-* custom start angle
+Demo of plotting a `~matplotlib.axes.Axes.pie` chart
+"""
 
-Note about the custom start angle:
+# %%
+# Label slices
+# ------------
+#
+# Plot a pie chart of animals and label the slices. To add
+# labels, pass a list of labels to the *labels* parameter
 
-The default ``startangle`` is 0, which would start the "Frogs" slice on the
-positive x-axis. This example sets ``startangle = 90`` such that everything is
-rotated counter-clockwise by 90 degrees, and the frog slice starts on the
-positive y-axis.
-"""
 import matplotlib.pyplot as plt
-
-# Pie chart, where the slices will be ordered and plotted counter-clockwise:
 labels = 'Frogs', 'Hogs', 'Dogs', 'Logs'
 sizes = [15, 30, 45, 10]
-explode = (0, 0.1, 0, 0)  # only "explode" the 2nd slice (i.e. 'Hogs')
 
-fig1, ax1 = plt.subplots()
-ax1.pie(sizes, explode=explode, labels=labels, autopct='%1.1f%%',
-        shadow=True, startangle=90)
-ax1.axis('equal')  # Equal aspect ratio ensures that pie is drawn as a circle.
+fig, ax = plt.subplots()
+ax.pie(sizes, labels=labels)
 
-plt.show()
+###############################################################################
+# Each slice of the pie chart is a `.patches.Wedge` object; therefore in
+# addition to the customizations shown here, each wedge can be customized as
```

Comment:
```suggestion
# addition to the customizations shown here, each wedge can be customized using the *wedgeprops* argument, as
```
to be clear that the nested pie example isn't specifically about customising wedges?